crates/credentials/src/vc.rs

```diff
@@ -6,16 +6,52 @@ use jwt::{
 };
 use serde::{Deserialize, Serialize};
 use std::{collections::HashMap, sync::Arc};
+use chrono::Utc;
```

Comment:
this made me think, we should come to a consensus on this https://github.com/TBD54566975/web5-rs/issues/183

what you have here is fine, not proposing a change, just raising awareness